src/resizable_array.jl

```diff
@@ -144,10 +144,64 @@ end
 
 Base.iterate(array::ResizableArray{T, V, N}, state = 1) where {T, V, N} = iterate(array.data, state)
 
+function Base.map(f, array::ResizableArray{T, V, N}) where {T, V, N}
+    result = map(f, array.data)
+    return ResizableArray(result)
+end
+
+__length(array::ResizableArray{T, V, N}) where {T, V, N} = length(array) == 0 ? 0 : __recursive_length(Val(N), array.data)
```

Comment:
Where `length(array::ResizableArray)` is defined and what's this function purpose? `__recursive_length()` already checks `length(array) == 0` on line 154.